docs/feature_space_shift_cadet.md

```diff
@@ -0,0 +1,26 @@
+## Space Cadet Shift Enter: The future, built in
+
+Based on the Space Cadet Shift by Steve Losh [described](http://stevelosh.com/blog/2012/10/a-modern-space-cadet/) 
+Essentially, you hit the Shift on its own, and it acts as the enter key. When hit with other keys, the Shift key keeps working as it always does. Yes, it's as cool as it sounds. This solution works better than using a macro since the timers defined in quantum allow us to tell when another key is pressed, rather than just having a janky timer than results in accidental endlines. 
+
+To use it, use `KC_SFTENT` (Shift, Enter) for any Shift on your keymap.
+
+It's defaulted to work on US keyboards, but if you'd like to use a different key for Enter, you can define those in your `config.h` like this:
+
+    #define SFTENT_KEY KC_ENT
+
+
+The only other thing you're going to want to do is create a `Makefile` in your keymap directory and set the following:
```

Comment:
`Makefiles` are actually ignored now! This will need to be the `rules.mk` file :)

Reply:
Ah, I changed my personal Makefiles to a rules.mk file, but I missed that in the documentation. Uploading the change to doc now. Thanks @jackhumbert!